modules/sdk/project-templates/project-templates-workspace/src/main/resources/archetype-resources/GETTING_STARTED.markdown

```diff
@@ -149,6 +149,18 @@ $h4 liferay.workspace.bundle.dist.include.metadata
 Set this to true to append metadata for the current environment settings and
 timestamp. The default value is `false`.
 
+$h4 liferay.workspace.docker.access.token
```

Comment:
change property to `liferay.workspace.docker.user.access.token`

this will move this property closer to `liferay.workspace.docker.username`